tutorials/hess.ipynb

```diff
@@ -72,12 +69,64 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_store = DataStore.from_file(\n",
-    "    \"$GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz\"\n",
-    ")\n",
-    "mask = data_store.obs_table[\"TARGET_NAME\"] == \"Crab\"\n",
-    "obs_table = data_store.obs_table[mask]\n",
-    "observations = data_store.get_observations(obs_table[\"OBS_ID\"])"
+    "config = \"\"\"\n",
+    "general:\n",
```

Comment:
Default logging should be fine. Remove from this first example people see? There config should be as simple as possible, no?